### PR TITLE
Add helm chart

### PR DESCRIPTION
**Pull request by pipo02mix**
Towards https://github.com/giantswarm/roadmap/issues/275

### CODE REVIEW

File: helm/goldilocks-app/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 apiVersion: v1
-appVersion: 2.2.0
+appVersion: v3.1.4
+dependencies:
+- condition: vpa.enabled
+  name: vpa
+  repository: https://charts.fairwinds.com/stable
```
Comment (yasn77) [inline helm/goldilocks-app/Chart.yaml:6]: We have VPA in our catalogue, I think we should reference that
(https://github.com/giantswarm/giantswarm-playground-catalog/blob/master/vertical-pod-autoscaler-app-1.0.3.tgz)

File: helm/goldilocks-app/charts/vpa/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
```
Comment (yasn77) [inline helm/goldilocks-app/charts/vpa/Chart.yaml:1]: Same as the previous comment, I don't think we need to ship it as a sub chart, we should just pull it in

File: helm/goldilocks-app/values.yaml [hunk 3]
```diff
@@ -1,5 +1,120 @@
...
-goldilocks:
-  image:
-    repository: quay.io/giantswarm/goldilocks
+vpa:
+  # vpa.enabled -- If true, the vpa will be installed as a sub-chart
+  enabled: false
+
+image:
+  # image.repository -- Repository for the goldilocks image
+  repository: quay.io/giantswarm/goldilocks
```
Comment (yasn77) [inline helm/goldilocks-app/values.yaml:10]: We have a pattern for images -> https://github.com/giantswarm/installations/blob/master/default/appcatalog/default-appcatalog-values.yaml#L21 

This ensures that China clusters use the right repo

**Review by yasn77 — CHANGES_REQUESTED** [review]
The main thing is, that I don't think we need to ship VPA as a sub chart, just use what we already have

**Comment by pipo02mix** [comment]
can you review @yasn77 ?

**Review by yasn77 — APPROVED** [review]
Thanks for the updates @pipo02mix 

LGTM :shipit:

**Comment by ubergesundheit** [comment]
Can you rename the jobs in the circle ci config file to include the catalog which they push to?

Then we can change the branch protection.

**Pull request merged**